documentation/modules/deploying/proc-deploying-kafkaconnector.adoc

```diff
@@ -35,11 +45,63 @@ spec:
     # ...
 ----
 +
-<1> Enter a name for the `KafkaConnector` resource. This will be used as the name of the connector within Kafka Connect. You can choose any name that is valid for a Kubernetes resource.
-<2> Enter the name of the Kafka Connect cluster in which to create the connector.
-<3> The name or alias of the connector class. This should be present in the image being used by the Kafka Connect cluster.
-<4> The maximum number of tasks that the connector can create.
-<5> Configuration settings for the connector. Available configuration options depend on the connector class.
+<1> Name of the `KafkaConnector` resource, which is used as the name of the connector. Use any name that is valid for a Kubernetes resource.
+<2> Name of the Kafka Connect cluster to create the connector in.
+<3> Full name or alias of the connector class. This should be present in the image being used by the Kafka Connect cluster.
+<4> Maximum number of Kafka Connect `Tasks` that the connector can create.
+<5> Connector configuration as key-value pairs.
++
+The `FileStreamSourceConnector` class accepts the following configuration options:
++
+[cols="4*",options="header",stripes="none",separator=¦]
```

Comment:
Did we say we should have table titles for accessibility?

Reply:
Yes, good spot.